app/build.gradle

```diff
@@ -0,0 +1,58 @@
+plugins {
+    id 'com.android.application'
+    id 'kotlin-android'
```

Comment:
many errors are related to this plugin. needs immediate resolution to continue with development